types/office-js-preview/index.d.ts

```diff
@@ -19815,826 +19758,4189 @@ declare namespace Excel {
+        * @remarks
+        * [Api set: 1.16]
+        */
+        arrowTrendingLines = "ArrowTrendingLines",
+        /**
+        * Art icon. Displays as a paint palette.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        art = "Art",
+        /**
+        * Atom icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        atom = "Atom",
+        /**
+        * Attach icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        attach = "Attach",
+        /**
+        * Automobile icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        automobile = "Automobile",
+        /**
+        * Autosum icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        autosum = "Autosum",
+        /**
+        * Backpack icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        backpack = "Backpack",
+        /**
+        * Badge icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        badge = "Badge",
+        /**
+        * Balloon icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        balloon = "Balloon",
+        /**
+        * Bank icon. Displays as a building with pillars and a triangular roof.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        bank = "Bank",
+        /**
+        * Barcode scanner icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        barcodeScanner = "BarcodeScanner",
+        /**
+        * Basketball icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        basketball = "Basketball",
+        /**
+        * Battery empty icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        battery0 = "Battery0",
+        /**
+        * Battery full icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        battery10 = "Battery10",
+        /**
+        * Beach icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        beach = "Beach",
+        /**
+        * Beaker icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        beaker = "Beaker",
+        /**
+        * Bed icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        bed = "Bed",
+        /**
+        * Bin full icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        binFull = "BinFull",
+        /**
+        * Bird icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        bird = "Bird",
+        /**
+        * Bluetooth icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        bluetooth = "Bluetooth",
+        /**
+        * Board icon.
+        *
+        * @remarks
+        * [Api set: 1.16]
+        */
+        board = "Board",
+        /**
+        * Board games icon.
```

Comment:
But which board games? Euro games or Ameri-trash? These are very important questions for no one but me.